internal/databases/mysql/stream_fetch_handler.go

```diff
@@ -45,9 +45,20 @@ func downloadAndDecompressStream(folder storage.Folder, fileName string) error {
 	if err != nil {
 		return err
 	}
+	binlogs, _, err := folder.GetSubFolder(BinlogPath).ListFolder()
+	if err != nil {
+		return err
+	}
+	var backupStartUploadTime time.Time
```

Comment:
This name is confusing, as it's binlog uploading datetime, and it happens after backup started, and even after backup finished.